nostrudel/umbrel-app.yml

```diff
@@ -0,0 +1,42 @@
+manifestVersion: 1
+id: nostrudel
+category: social
+name: noStrudel
+version: "0.37.1"
+tagline: A sandbox for exploring nostr
+icon: https://raw.githubusercontent.com/hzrd149/umbrel-community-app-store/master/hzrd149-nostrudel/icon.svg
+description: >-
+  noStrudel is a nostr web client that is focused on exploring the nostr network.
+
+  The focus is on power users or any other users who want to learn the inner workings of nostr.
+releaseNotes: >-
+  Changes:
+
+   - Rebuild DMs view to be more readable
+
+   - Show reactions and zaps on DMs
+
+   - Add support for threads in DMs
+
+   - Add support for Amber signer
+
+   - Add support for NIP-46 signer
+
+   - Support NIP-31 on unknown event kinds
+
+developer: hzrd149
+website: https://github.com/hzrd149/nostrudel
+dependencies: []
+repo: https://github.com/hzrd149/nostrudel
+support: https://github.com/hzrd149/nostrudel/issues
+port: 3149
+gallery:
+  - https://raw.githubusercontent.com/hzrd149/nostrudel/v0.37.1/screenshots/profile.png
+  - https://raw.githubusercontent.com/hzrd149/nostrudel/v0.37.1/screenshots/community.png
+  - https://raw.githubusercontent.com/hzrd149/nostrudel/v0.37.1/screenshots/photography.png
+  - https://raw.githubusercontent.com/hzrd149/nostrudel/v0.37.1/screenshots/streaming2.png
```

Comment:
These can be changed to 1.jpg, 2.jpg, etc. For example:

https://github.com/getumbrel/umbrel-apps/blob/f6eae6fdc53a1bd71528fed49118e299da5d1aea/mempool/umbrel-app.yml#L25-L28

We will host the gallery images here as .jpgs:
https://github.com/getumbrel/umbrel-apps-gallery

Reply:
updated :+1: